express-notes-api/server.ts

```diff
@@ -135,20 +142,10 @@ app.listen(8080, () => {
 });
 
 async function readJSON() {
-  try {
-    data = JSON.parse(await readFile('data.json', 'utf8'));
-  } catch (err) {
-    console.log(err);
-  }
+  const data: Data = JSON.parse(await readFile('data.json', 'utf8'));
+  return data;
```

Comment:
You can actually combine these into one statement and still get type checking in one of two ways. I'll give you my preferred way first:
```
async function readJSON(): Promise<Data> {
  return JSON.parse(await readFile('data.json', 'utf8'));
}
```
That is by far the best way because your function declares its return type. You should always declare the return type on your functions!

This next way uses a type assertion:
```
async function readJSON() {
  return JSON.parse(await readFile('data.json', 'utf8')) as Data;
}
```